packages/ref/src/cmip_ref/datasets/obs4mips.py

```diff
@@ -106,8 +110,13 @@ def parse_obs4mips(file: str) -> dict[str, Any | None]:
         return info
 
     except TypeError:
+        logger.warning(traceback_message)
+        return {"INVALID_ASSET": file, "TRACEBACK": traceback_message}
+    except AttributeError:
+        logger.warning(traceback_message)
```

Comment:
You can catch multiple exceptions in the sample block

```suggestion
    except TypeError, AttributeError:
        logger.warning(traceback_message)
```